packages/language-server/test/plugins/svelte/features/getHoverInfo.test.ts

```diff
@@ -66,7 +66,7 @@ describe('SveltePlugin#getHoverInfo', () => {
         });
 
     describe('should return hover for #', () => {
-        (['if', 'each', 'await'] as const).forEach((tag) => {
+        (['if', 'each', 'await :then', 'await then'] as const).forEach((tag) => {
```

Comment:
I would leave the test as-is, I think it's unrelated and your changes should not affect this. Instead you need to adjust the method signature in `getCompletions`.

Reply:
what's the alternative?

Reply:
Putting two different completions with the same label would be a bad idea.

Reply:
These tests are for hover info, not completions.

Reply:
oh right, I'm stupid